### PR TITLE
varnish: allow proxying beta

### PR DESCRIPTION
**Pull request by RhinosF1**

### CODE REVIEW

File: modules/varnish/templates/default.vcl [hunk 1]
```diff
@@ -529,6 +529,11 @@ sub vcl_recv {
 		}
 	}
 
+        if (req.http.Host ~ "^*.betaheze.org") {
```
Comment (JohnFLewis) [inline modules/varnish/templates/default.vcl:532]: This is invalid regex

**Review by JohnFLewis — CHANGES_REQUESTED** [review]

File: modules/varnish/templates/default.vcl (hunk 1 above)
Comment (Universal-Omega) [reply modules/varnish/templates/default.vcl:532]: This should be `^(.*\.)?betaheze\.org` in order to also do betaheze.org itself, or `^.*\.betaheze\.org` for just subdomains. I'm not certain if the periods need escaped or not though.

Comment (JohnFLewis) [reply modules/varnish/templates/default.vcl:532]: > I'm not certain if the periods need escaped or not though.

They don't *strictly* need to be, but should be.

**Comment by Universal-Omega** [comment]
Conflicts

File: modules/varnish/templates/default.vcl [hunk 2]
```diff
@@ -263,6 +263,11 @@ sub vcl_recv {
 		}
 	}
 
+  if (req.http.Host ~ "^(.*\.)?betaheze\.org") {
```
Comment (Universal-Omega) [inline modules/varnish/templates/default.vcl:266]: Tabbing

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

**Review by Reception123 — APPROVED** [review]

**Pull request merged**